### ISSUE TITLE
Honeywell T6 Pro Z-wave shows reversed cool/heat setpoints in HA Thermostat

### ISSUE THREAD

**Issue by clboles** [jason0x43/hacs-hubitat#73]
I've just moved from Nest to Honeywell T6 Pro Z-wave thermostats.  The states are reported properly in Hubitat, but when I try to use the Thermostat card in Home Assistant, the cool and heat setpoints are reversed, causing the card to not work very well.

Here is the state info from Hubitat:

```
battery : 95
coolingSetpoint : 73.0
heatingSetpoint : 70.0
humidity : 55
temperature : 73.0
thermostatFanMode : auto
thermostatMode : auto
thermostatOperatingState : idle
thermostatSetpoint : 73.0
```

And here are the state attributes from Home Assistant:

```
hvac_modes:
  - heat
  - heat_cool
  - cool
  - 'off'
min_temp: 45
max_temp: 95
fan_modes:
  - 'on'
  - auto
preset_modes:
  - home
  - away
current_temperature: 73
temperature: null
target_temp_high: 70
target_temp_low: 73
current_humidity: 55
fan_mode: auto
hvac_action: idle
preset_mode: home
friendly_name: Master Bedroom Thermostat
supported_features: 27
```

Notice target_temp_high is the heat value, and target_temp_low is the cool value.

Is there any additional info I can provide to help debug this?

Thanks!

**Comment by clboles** [comment]
In climate.py, perhaps this is where the swap is happening:

```
    async def async_set_temperature(self, **kwargs: Any) -> None:
        """Set new target temperature."""
        if self.hvac_mode == HVAC_MODE_HEAT_COOL:
            temp_low = kwargs.get(ATTR_TARGET_TEMP_LOW)
            temp_high = kwargs.get(ATTR_TARGET_TEMP_HIGH)
            **if temp_low is not None:
                await self.send_command(CMD_SET_COOLING_SETPOINT, temp_low)
            if temp_high is not None:
                await self.send_command(CMD_SET_HEATING_SETPOINT, temp_high)**
        else:
            temp = kwargs.get(ATTR_TEMPERATURE)
            if temp is not None:
                if self.hvac_mode == HVAC_MODE_COOL:
                    await self.send_command(CMD_SET_COOLING_SETPOINT, temp)
                elif self.hvac_mode == HVAC_MODE_HEAT:
                    await self.send_command(CMD_SET_HEATING_SETPOINT, temp)
```

**Comment by jason0x43** [comment]
Resolved by #74
